cmake/SetupXsimd.cmake

```diff
@@ -6,6 +6,10 @@ option(USE_XSIMD "Use xsimd if it is available" OFF)
 if(USE_XSIMD)
   find_package(xsimd REQUIRED)
 
+  if(xsimd_VERSION VERSION_LESS 11.0.1)
+    message(FATAL_ERROR "xsimd must be at least 11.0.1, got ${xsimd_VERSION}")
+  endif()
```

Comment:
`find_package(xsimd REQUIRED 11.0.1)`

Reply:
I did not have this work correctly for xsimd. Feel free to give it a try later, but I spent enough time with it that I can't put more in